### PR TITLE
Vtexplain no globals

### PR DESCRIPTION
**Pull request by systay**
## Description
Minor refactoring of vtexplain, removing global variables that did not need to be global.

## Checklist
-   [x] "Backport me!" label has been added if this change should be backported
-   [x] Tests were added or are not required
-   [x] Documentation was added or is not required

### CODE REVIEW

File: go/cmd/vtexplain/vtexplain.go [hunk 1]
```diff
@@ -113,7 +113,6 @@ func getFileParam(flag, flagFile, name string, required bool) (string, error) {
 }
 
 func main() {
-	defer vtexplain.Stop()
```
Comment (GuptaManan100) [inline go/cmd/vtexplain/vtexplain.go:116]: Shouldn't we call vte.Stop() or some other equivalent of this in the end?

**Review by GuptaManan100 — APPROVED** [review]

File: go/cmd/vtexplain/vtexplain.go (hunk 1 above)
Comment (systay) [reply go/cmd/vtexplain/vtexplain.go:116]: We are. I just moved the defer to after we have done the Init

**Comment by ajm188** [comment]
as a followup, maybe `NewExecutor` is now a better name than `Init` for that function?

**Comment by systay** [comment]
@ajm188 sounds good to me

**Pull request merged**